### ISSUE TITLE
[Woo POS] Add missing lower priority transitions

### ISSUE THREAD

**Issue by staskus** [woocommerce/woocommerce-ios#13721]
A list of missing transitions in POS.

Time-box a specific amount of time and tackle as many as possible. I suggest prioritizing them by how often they are likely to be encountered. 

These are higher priority transitions: https://github.com/woocommerce/woocommerce-ios/issues/13700

## Happy Path

p91TBi-bHy-p2

#### Initial loading 
- [ ] Loading View pushed from the bottom [0:24]
- [x] POS Dashboard pushed from the top [0:26]

#### Simple Products
- [ ] ‘Simple products only’  message dismissal fade out , item list pushed up [1:30]
- [ ] ‘Simple products only’ modal fade in transition from the bottom [1:35]
- [ ] ‘Simple products only’ modal fade-out transition to the bottom [1:43]

#### Floating button
- [ ] Floating Button transitions from `Reader Disconnected` to `Reader Connected` by animating resizing of the button

## Unhappy Path

p91TBi-bHy-p2

### Initial loading
- [ ] ‘No supported products found’ found’ pushed from the top [1:52]
- [ ] ‘Error loading products’ pushed from top [2:43]
- [ ] ‘No supported products found’ / ‘Error loading products’ pushed to top when retrying [2:57]

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by malinajirka** [comment]
@staskus Is this still valid? Do you believe it's worth spending time on compared to other available tasks?

**Comment by staskus** [comment]
@malinajirka no, I think it's outdated and we haven't found a need for additional ones.

I think any new animation tasks could be created out of design review/feedback.
